source/utils/network.ts

```diff
@@ -0,0 +1,12 @@
+import store from 'state/store';
+
+export const isActiveNetwork = (chain: string, chainId: number) => {
+  const { activeNetwork } = store.getState().vault;
+  const isSysCore = activeNetwork.url.includes('blockbook');
```

Comment:
check for chain as well, we can't tell if it is a syscoin core network or evm just by the url

Reply:
This is still a terrible way of doing it. I just copied the same solution that i saw somewhere else. Will update